client/src/components/chat/Chat.js

```diff
@@ -161,14 +165,17 @@ class Chat extends Component {
     const {
       disabled,
       handleChange,
-      selectedProposal,
```

Comment:
The addition of `selectedProposal` is a newer change. It should not have been deleted.